frontend/components/MainPage/sections/GasBalanceSection.tsx

```diff
@@ -126,9 +126,9 @@ export const GasBalanceSection = () => {
                 <a
                   href={
                     `${
-                      EXPLORER_URL[
+                      EXPLORER_URL_BY_MIDDLEWARE_CHAIN[
                         // TODO: fix unknown
-                        homeChainId as unknown as keyof typeof EXPLORER_URL
+                        homeChainId as unknown as keyof typeof EXPLORER_URL_BY_MIDDLEWARE_CHAIN
```

Comment:
should probably do `asMiddlewareChain(homeChainId)` instead of aliases
or maybe can use `EXPLORER_URL_BY_EVM_CHAIN_ID`?